src/renderer/components/table/table.tsx

```diff
@@ -136,14 +135,50 @@ export class Table extends React.Component<TableProps> {
     return headElem;
   }
 
-  getSorted(items: any[]) {
-    const { sortBy, orderBy: order } = this.sortParams;
-    const sortingCallback = this.props.sortable[sortBy] || noop;
+  getSorted(rawItems: ItemObject[]) {
```

Comment:
This method is now complex enough that we really need tests to confirm it works.